tests/unit/test_exceptions.py

```diff
@@ -355,7 +355,7 @@ def test_error_details_from_grpc_response():
     status.details.append(status_br_detail)
     status.details.append(status_ei_detail)
 
-    # Actualy error doesn't matter as long as its grpc.Call,
+    # Actually error doesn't matter as long as its grpc.Call,
```

Comment:
Should this be `Actual error doesn't matter...`?

Reply:
Fixed